### PR TITLE
LOOP-2974 Support pod time zone change

### PR DESCRIPTION
**Pull request by ps2**
https://tidepool.atlassian.net/browse/LOOP-2974

### CODE REVIEW

**Review by darinkrauss — APPROVED** [review]
LGTM! 🚀

File: Loop/Managers/Alerts/InAppModalAlertIssuer.swift [hunk 1]
```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2020 LoopKit Authors. All rights reserved.
 //
 
-import Foundation
+import UIKit
```
Comment (rickpasetto) [inline Loop/Managers/Alerts/InAppModalAlertIssuer.swift:9]: [nit] I'm not sure I see why this change was necessary (?)

File: Loop/Managers/Alerts/AlertManager.swift [hunk 2]
```diff
@@ -7,8 +7,9 @@
 //
 
 import LoopKit
+import UIKit
```
Comment (rickpasetto) [inline Loop/Managers/Alerts/AlertManager.swift:10]: [nit] I'm not sure I see why this change was necessary (?)

File: Loop/Managers/Alerts/UserNotificationAlertIssuer.swift [hunk 3]
```diff
@@ -7,6 +7,7 @@
 //
 
 import LoopKit
+import UIKit
```
Comment (rickpasetto) [inline Loop/Managers/Alerts/UserNotificationAlertIssuer.swift:10]: [nit] I'm not sure I see why this change was necessary (?)

File: Loop/Managers/CriticalEventLogExportManager.swift [hunk 4]
```diff
@@ -7,7 +7,7 @@
 //
 
 import os.log
-import Foundation
+import UIKit
```
Comment (rickpasetto) [inline Loop/Managers/CriticalEventLogExportManager.swift:10]: [nit] I'm not sure I see why this change was necessary (?)

File: Loop/Models/LoopConstants.swift [hunk 5]
```diff
@@ -9,6 +9,7 @@
 import Foundation
 import HealthKit
 import LoopKit
+import UIKit
```
Comment (rickpasetto) [inline Loop/Models/LoopConstants.swift:12]: [nit] I'm not sure I see why this change was necessary (?)

**Review by rickpasetto — APPROVED** [review]
Just a few nits regarding why `import UIKit` alone was added, but other than that LGTM.

File: Loop/Managers/Alerts/InAppModalAlertIssuer.swift (hunk 1 above)
Comment (ps2) [reply Loop/Managers/Alerts/InAppModalAlertIssuer.swift:9]: Same answer for this and the other comments below: LoopKit was including UIKit in its umbrella header, which I removed. These files use various symbols from UIKit, and so it needs to be imported explicitly.
